components/webcompat_exceptions/features.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_WEBCOMPAT_EXCEPTIONS_FEATURES_H_
+#define BRAVE_COMPONENTS_WEBCOMPAT_EXCEPTIONS_FEATURES_H_
+
+#include "base/feature_list.h"
+
+namespace webcompat {
```

Comment:
this should match the directory name, mb rename the component?

Reply:
done